salt/hubblestack/pulsar_dependency.sls

```diff
@@ -0,0 +1,3 @@
+install_pulsar_dependency:
+  pkg.installed:
+    - name: python-pyinotify
```

Comment:
The install_pip state is run on all of the minions, but adding the pyinotify package to pillar data can be done on specific minions that we are targeting, which would only install it on those minions that have it in the pillar data.